src/controllers/signup.controller.js

```diff
@@ -0,0 +1,36 @@
+import SignupService from '../services/signup.service';
+import passwordHandler from '../services/bcrypt';
```

Comment:
The filename should be `bcrypt.service.js`. Also, the import name must match the exported default name. `bcryptService` is more succinct as bcrypt is not specific to handling password but it's a hashing package.

Reply:
@higustave-ops Not sure you implemented the feedback properly. The class name should be `BcryptService`